### PR TITLE
Private network page

### PR DESCRIPTION
**Pull request by jmcook1186**
This PR removes the existing "Getting Started > Private network tutorial" page and merges the relevant information into the "Using Geth > Private Networks" page. The step-by-step tutorial has been rewritten from scratch.

### CODE REVIEW

File: docs/_interface/Private-Network.md [hunk 1]
```diff
@@ -3,103 +3,84 @@ title: Private Networks
 sort_key: B
 ---
 
-This guide explains how to set up a private network of multiple Geth nodes. An Ethereum
-network is a private network if the nodes are not connected to the main network. In this
-context private only means reserved or isolated, rather than protected or secure.
+This guide explains how to set up a private network of multiple Geth nodes. An Ethereum network is private if the nodes are not connected to the main network. In this
+context private only means reserved or isolated, rather than protected or secure. A fully controlled, private Ethereum network is useful as a backend for network integration testing for core developers working on issues realting to networking/blockchain syncing etc. Private networks are also useful for Dapp developers testing multi-block and multi-user scenarios.
```
Comment (fjl) [inline docs/_interface/Private-Network.md:7]: `A fully controlled, private...`

this sentence is too long.

Comment (jmcook1186) [reply docs/_interface/Private-Network.md]: ok, shortened.

File: docs/_interface/Private-Network.md [hunk 2]
```diff
@@ -3,103 +3,84 @@ title: Private Networks
...
+
+
+## Prerequisites
+
+To follow the tutorial on this page it is necessary to have a working Geth installation (instructions [here](../../_site/docs/install-and-build/installing-geth.html)). It is also helpful to understand Geth fundamentals (see [Getting Started](../../docs/_getting-started/index.md)).
+
+
+## Private Networks
+
+A private network is composed of multiple Ethereum nodes that are all run on the local machine. In order to run multiple nodes locally, each one requires a separate data directory (`--datadir`), a different `eth` and `http` port and a unique `ipc` endpoint. The nodes must also know about each other and be able to exchange information, share an initial state and a common consensus algorithm. The remainder of this page will explain how to configure Geth so that these basic requirements are met, enabling a private network to be started.
```
Comment (holiman) [inline docs/_interface/Private-Network.md:19]: > A private network is composed of multiple Ethereum nodes that are all run on the local machine.

A private network does not need to be run all off one local machine. It's pretty common to run a 'private network' across a set of hosts, e.g. for hackathons

Comment (holiman) [reply docs/_interface/Private-Network.md:19]: > a different `eth` and `http` port and a unique `ipc` endpoint.

The `eth` is simply `--port`. And re ipc endpoint, doesn' that follow along with the datadir? So if you set the datadir, the ipc will be also be changed accordingly

Comment (jmcook1186) [reply docs/_interface/Private-Network.md:19]: yes, that is right. This sentence was inherited from the original doc, I thought it was useful to point out which specific things should differ between the nodes, even if they don't have to be explicitly defined. But on reflection it is a bit confusing - will change.

**Comment by jmcook1186** [comment]
rebased onto main to resolve merge conflicts

File: docs/_interface/Private-Network.md [hunk 3]
```diff
@@ -153,146 +119,353 @@ adapt to the amount of mining resources you deploy on the chain.
...
-### Ethash: Running A Miner
+### Running A Miner (Ethash)
 
-For proof-of-work in a simple private network, a single CPU miner instance is enough to
-create a stable stream of blocks at regular intervals. To start a Geth instance for
-mining, run it with all the usual flags and add the following to configure mining:
+For PoW in a simple private network, a single CPU miner instance is enough to create a stable stream of blocks at regular intervals. To start a Geth instance for mining, it can be run with all the usual flags plus the following to configure mining:
 
 ```shell
 geth <other-flags> --mine --miner.threads=1 --miner.etherbase=0x0000000000000000000000000000000000000000
```
Comment (s1na) [inline docs/_interface/Private-Network.md:223]: Can you please use a different account here to address this issue https://github.com/ethereum/go-ethereum/pull/25106? maybe one of the alloc accounts

Comment (jmcook1186) [reply docs/_interface/Private-Network.md]: sure - done.

**Comment by s1na** [comment]
Overall it looks good to me (just pushed some fixes like typos and broken links). I did have some thoughts re private networks that could be worked on in future:

- It seems to me that the private networking page in "getting started" was more about the networking aspect. I.e. how to have 2 or more nodes talk to each other, e.g. by manually `addPeer`, `removePeer` etc. and easily forming clusters using `bootnode` cmd. I feel like that was mashed in together with the creating a private chain and it could be fleshed out more independently.
- There's the tool `puppet` which makes creating a new private chain/network pretty easy. Room for documenting that.

Going ahead and merging this now.

**Pull request merged**